default.hbs

```diff
@@ -16,8 +16,14 @@
   <footer class="site-footer">
     <a class="subscribe" href="{{@blog.url}}/rss/">Subscribe!</a>
     <div class="inner">
-       <section class="copyright">All content copyright <a href="{{@blog.url}}/">{{@blog.title}}</a> &copy; {{date format="YYYY"}} &bull; All rights reserved.</section>
-       <section class="poweredby">Proudly published with <a href="https://ghost.org">Ghost</a></section>
+       <section class="copyright">
+          All content copyright
+          <a href="{{@blog.url}}/">{{@blog.title}}</a>
+          &copy; {{date format="YYYY"}} &bull; All rights reserved.
+      </section>
```

Comment:
Looks like indentation might be a bit off on lines above this closing `</section>`?